content/code-security/security-advisories/publishing-a-security-advisory.md

```diff
@@ -92,6 +76,22 @@ Publishing a security advisory deletes the temporary private fork for the securi
 
 {% data reusables.repositories.github-reviews-security-advisories %}
 
+## Requesting a CVE identification number (Optional)
+
+Github can provide a CVE number if needed and if the software is not under the scope of another CVE Numbering Authority (CNA). Anyone with admin permissions to a security advisory can request a CVE identification number for the security advisory. If the advisory already has a CVE number, then that can be input in the advisory form.
```

Comment:
👋 @darakian, thanks heaps for this PR! I agree that if the step is optional, then it should be moved lower and marked as such. 

🤔 Most of the added content in this paragraph is already provided in the next paragraph in the `{% data reusables.repositories.request-security-advisory-cve-id %}` reusable. You can see the content of that resuable file [here](https://github.com/github/docs/blob/d1049f143a59a616a834cde9ea27722d6fcda77c/data/reusables/repositories/request-security-advisory-cve-id.md). You can see a [built stage of this branch here](https://docs-10210--reorder-advisory-p.herokuapp.com/en/code-security/security-advisories/publishing-a-security-advisory#requesting-a-cve-identification-number-optional).

I would recommend sticking with the current wording in the reusable, or modifying the reusable directly.

```suggestion
Anyone with admin permissions to a security advisory can request a CVE identification number for the security advisory.
```

Reply:
Thanks for the tip. I was unaware of what those `{%...%}` patterns were. I've pushed a commit which removes the leading text from the base doc and alters the reusable. Let me know what you think @lucascosti 

@rschultheis can I get a second review on the updated content?